Example/Tests/BPKPanelSnapshotTest.m

```diff
@@ -0,0 +1,72 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#import <Backpack/Panel.h>
+#import <Backpack/Color.h>
+
+@interface BPKPanelSnapshotTest : FBSnapshotTestCase
+
+@end
+
+@implementation BPKPanelSnapshotTest
+
+- (void)setUp {
+    [super setUp];
+    self.recordMode = NO;
+}
+
+- (void)configureParentView:(UIView *) parentView :(BPKPanel *) panel {
```

Comment:
Don't use spaces like you have in this signature instead write it like this

```objective-c
- (void)configureParentView:(UIView *)parentView forPanel:(BPKPanel *)panel
```

Reply:
Didn't realise internal and external arg names were also in Objective-C, I like them in Swift.